book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] Kindly consider adding a method `add_rental` in this class after initialize method. In that method you will be getting the `person,date` and inside that method, you will be initializing  and returning a rental object where you will be also giving the self `book class` to make the association right. example: `Rental.new(date, self, person)`